### PR TITLE
Add multi-schema support

### PR DESCRIPTION
**Pull request by unclecheese**
Resolves: https://github.com/silverstripe/silverstripe-graphql/issues/158

### CODE REVIEW

File: _config/graphql.yml [hunk 1]
```diff
@@ -1,16 +1,17 @@
 ---
 Name: cmsgraphql
```
Comment (maxime-rainville) [inline _config/graphql.yml]: Can we had a bit of conditional logic here just to be on the safe side?
```yml
---
Name: cmsgraphql
Only:
  moduleexists: silverstripe/graphql
---
```

**Comment by maxime-rainville** [comment]
silverstripe/silverstripe-graphql#173 and silverstripe/recipe-cms#18 need to be merged and the travis build needs to be restarted.

**Comment by robbieaverill** [comment]
Dependencies merged and build restarted

**Comment by maxime-rainville** [comment]
Build is failing because `silverstripe/asset-admin` still wants to install `silverstripe/graphql` v2. Once https://github.com/silverstripe/silverstripe-asset-admin/pull/806 is merged, test should go green.

**Pull request merged**